nikola/__main__.py

```diff
@@ -238,16 +239,16 @@ def __init__(self, nikola, quiet=False):
         self.quiet = quiet
 
     def load_tasks(self, cmd, opt_values, pos_args):
+        if 'DOIT_CONFIG' in config:
+            DOIT_CONFIG = config['DOIT_CONFIG']
+        else:
+            DOIT_CONFIG = {}
```

Comment:
Indeed it is.  I have this habit of writing all of my code like I'm preparing to show it in my CS1 course.